training/model_evaluation/environment/context/conda.yaml

```diff
@@ -21,8 +21,9 @@ dependencies:
   - azureml-telemetry=={{latest-pypi-version}}
   - azureml-evaluate-mlflow=={{latest-pypi-version}}
   - azureml-metrics=={{latest-pypi-version}}
+  - azureml-acft-common-components["image"]=={{latest-pypi-version}}
```

Comment:
Passing this was causing env build failures. Yesterday I was trying to deploy the updated env and it was failing because of this. 

If I moved the installation azureml-acft-common-components["image"] to Dockerfile as "RUN pip install azureml-acft-common-components["image"]" it passed. 

Can you take a look at this and confirm on your end as well? @MadhuM02 
CC: @rjaincc

Reply:
Yes @ankushbhatia2. I observed the same today while trying to build the dev environment. Guess packages with extra requires can only be installed through docker.